### ISSUE TITLE
[Checkbox] Color specs update

### ISSUE THREAD

**Issue by aweell** [dxc-technology/halstack-style-guide#297]
## Story description

- [ ] Update component documentation page with corrected color specs

## React

```
backgroundColorChecked: globalTokens.mediumBlue,
backgroundHoverColorChecked: "#7BB0D6",
backgroundColorCheckedOnDark: "#FFFFFF",
backgroundHoverColorCheckedOnDark: "#B1B1B1",
borderColor: globalTokens.mediumBlue,
borderHoverColor: "#7BB0D6",
borderColorOnDark: "#FFFFFF",
borderHoverColorOnDark: "#B1B1B1",
checkColor: globalTokens.white,
checkColorOnDark: "#0067B3",
fontColor: globalTokens.inherit,
fontColorOnDark: "#FFFFFF",
disabledBackgroundColorChecked: globalTokens.softBlue,
disabledBackgroundColorCheckedOnDark: globalTokens.softBlue,
disabledBorderColor: globalTokens.softBlue,
disabledBorderColorOnDark: globalTokens.softBlue,
disabledColorOnDark: "#BFBFBF",
disabledCheckColor: globalTokens.white,
disabledCheckColorOnDark: "#FFFFFF",
disabledFontColor: globalTokens.lighterBlack,
disabledFontColorOnDark: "#575757",
focusColor: globalTokens.blue,
```

## Angular **(Pending update)**

```
"--checkbox-borderColor": globalTokens.mediumBlue,
"--checkbox-checkColor": globalTokens.white,
"--checkbox-backgroundColorChecked": globalTokens.mediumBlue,
"--checkbox-fontColor": globalTokens.inherit,
"--checkbox-focusColor": globalTokens.blue,
"--checkbox-disabledBackgroundColorChecked": globalTokens.softBlue,
"--checkbox-disabledBorderColor": globalTokens.softBlue,
"--checkbox-disabledCheckColor": globalTokens.white,
"--checkbox-disabledFontColor": globalTokens.lighterBlack,
```

### On-light

|   Component token                         |   Element                                         |   Core token                                   |   Value             |
| ------------------------------------ | --------------------------------- | -------------------------------- | ------------- |
|   `backgroundColorChecked`            |   Fill                 |   `Hal-Blue-S-35`                          |   `#0067b3`   |
|   `hoverBackgroundColorChecked`  |   Fill:hover      |   `Hal-Blue-L-45`                          |   `#0086e6`   |
|   `disabledBackgroundColorChecked`            |   Fill:disabled                |   `Hal-Grey-L-75`       |   `#bfbfbf`   |
|   `borderColor`  |   Border      |   `Hal-Blue-S-35`                          |   `#0067b3`   |
|   `hoverBorderColor`  |   Border:hover      |   `Hal-Blue-L-45`                          |   `#0086e6`   |
|   `disabledBorderColor`  |   Border:disabled      |   `Hal-Grey-L-95`             |   `#f2f2f2`   |
|   `checkColor`  |   Check mark      |   `Hal-White`             |   `#ffffff`   |
|   `disabledCheckColor`  |   Check mark:disabled      |   `Hal-White`             |   `#ffffff`   |
|   `fontColor`  |   Label      |   `Hal-Black`             |   `#000000`   |
|   `disabledFontColor`  |   Label:disabled     |   `Hal-Grey-L-60`             |   `#999999`   |
|   `focusColor`  |   Outline:focus    |   `Hal-Blue-S-35`                          |   `#0067b3`   |

### On-dark

|   Component token                         |   Element                                         |   Core token                                   |   Value             |
| ------------------------------------ | --------------------------------- | -------------------------------- | ------------- |
|   `backgroundColorCheckedOnDark`  |   Fill      | `Hal-Blue-L-50`                          |   `#0095ff`   |
|   `hoverBackgroundColorCheckedOnDark`  |   Fill:hover      |   `Hal-Blue-S-35`                          |   `#0067b3`   |
|   `disabledBackgroundColorCheckedOnDark`            |   Fill:disabled                |   `Hal-Grey-S-40`       |   `#666666`   |
|   `borderColorOnDark`  |   Border      | `Hal-Blue-L-50`                          |   `#0095ff`   |
|   `hoverBorderColorOnDark`  |   Border:hover      | `Hal-Blue-S-35`                          |   `#0067b3`   |
|   `disabledBorderColorOnDark`  |   Border:disabled      |   `Hal-Grey-S-40`             |   `#666666`   |
|   `checkColorOnDark`  |   Check mark      |  `Hal-White`             |   `#ffffff`   |
|   `disabledCheckOnDark`  |   Check mark:disabled      |   `Hal-White`             |   `#ffffff`   |
|   `fontColorOnDark`  |   Label      |   `Hal-White`             |   `#ffffff`   |
|   `disabledFontColorOnDark`  |   Label:disabled     |   `Hal-Grey-L-40`             |   `#666666`   |
|   `focusColorOnDark`  |   Outline:focus     |    `Hal-Blue-L-50`                          |   `#0095ff`   |

**Comment by GauravMaurya0** [comment]
React
  backgroundColorChecked: globalTokens.mediumBlue,
  **backgroundHoverColorChecked: "#7BB0D6",
  backgroundColorCheckedOnDark: "#FFFFFF",
  backgroundHoverColorCheckedOnDark: "#B1B1B1"**,
  borderColor: globalTokens.mediumBlue,
  **borderHoverColor: "#7BB0D6",
  borderColorOnDark: "#FFFFFF",
  borderHoverColorOnDark: "#B1B1B1",**
  checkColor: globalTokens.white,
  **checkColorOnDark: "#0067B3",**
  fontColor: globalTokens.inherit,
  **fontColorOnDark: "#FFFFFF",**
  disabledBackgroundColorChecked: globalTokens.softBlue,
  **disabledBackgroundColorCheckedOnDark: globalTokens.softBlue,**
  disabledBorderColor: globalTokens.softBlue,
  ****disabledBorderColorOnDark: globalTokens.softBlue,**
  disabledColorOnDark: "#BFBFBF",**
  disabledCheckColor: globalTokens.white,
  **disabledCheckColorOnDark: "#FFFFFF",**
  disabledFontColor: globalTokens.lighterBlack,
  **disabledFontColorOnDark: "#575757",**
  focusColor: globalTokens.blue,
  
Angular
  "--checkbox-borderColor": globalTokens.mediumBlue,
  "--checkbox-checkColor": globalTokens.white,
  "--checkbox-backgroundColorChecked": globalTokens.mediumBlue,
  "--checkbox-fontColor": globalTokens.inherit,
  "--checkbox-focusColor": globalTokens.blue,
  "--checkbox-disabledBackgroundColorChecked": globalTokens.softBlue,
  "--checkbox-disabledBorderColor": globalTokens.softBlue,
  "--checkbox-disabledCheckColor": globalTokens.white,
  "--checkbox-disabledFontColor": globalTokens.lighterBlack, 

Angular don’t have certain tokens which is highlighted in React’s tokens

## Colors specs table

| Component token                         | Element       | Core token | Value |
| -----------------------------   | ----------- | -------- | --------- |
| `checkColor` | Caret icon | Hal-White | `#ffffff` |
| `checkColorOnDark` | Caret iconOnDark | - | - |
| `disabledCheckColor` | Caret icon:disabled | Hal-White | `#ffffff` |
| `disabledCheckColorOnDark` | Caret icon:disabledOnDark | - | - |
| `fontColor` | Label | Inherit | - |
| `fontColorOnDark` | Label:OnDark | `Hal-White` | `#ffffff` |
| `focusColor` | Border:focus | `Hal-Blue-S-35` | `#0067b3` |
| `borderColor` | Separator | `Hal-Blue-L-50` | `#0095ff` |
| `backgroundColor` | Container background | `Hal-Blue-L-50` | `#0095ff` |
| `disabledBackgroundColor` | Container background:disabled | `Hal-Blue-D-20` | `#003c66` |
| `disabledBackgroundColorOnDark` | Container background:disabledOnDark | `Hal-Blue-D-20` | `#003c66` |
| `disabledBorderColor` | Separator:disabled | `Hal-Blue-D-20` | `#003c66` |
| `disabledBorderColorOnDark` | Separator:disabledOnDark | `Hal-Blue-D-20` | `#003c66` |
| `disabledFontColor` | Label:disabled | `Hal-Grey-L-75` | `#bfbfbf` |
| `disabledFontColorOnDark` | Label:disabledOnDark | `Hal-Grey-S-40` | `#666666` |
| `backgroundHoverColor` | Container background:hover | `Hal-Blue-L-80` | `#99d5ff` |
| `backgroundColorCheckedOnDark` | Container background: OnDark | `Hal-White` | `#ffffff` |
| `backgroundHoverColorCheckedOnDark` | Container background:hoverOnDark | `Hal-Grey-L-75` | `#bfbfbf` |
| `borderHoverColor` | Separator:hover | `Hal-Blue-L-50` | `#0095ff` |
| `borderColorOnDark` | Separator:OnDark | `Hal-White` | `#ffffff` |
| `borderHoverColorOnDark` | Separator:hoverOnDark | `Hal-Grey-L-75` | `#bfbfbf` |
| `disabledColorOnDark` | Custom icon | `Hal-Grey-L-75` | `#bfbfbf` |
| `focusColor` | Custom icon:focus | `Hal-Blue-D-20` | `#003c66` |

**Comment by GomezIvann** [comment]
Changes made on the Accordion advanced tokens:

- [ ] `focusColorOnDark` token added.
- [ ] `backgroundHoverColorChecked` renamed as `hoverBackgroundColorChecked`.
- [ ] `backgroundHoverColorCheckedOnDark` renamed as `hoverBackgroundColorCheckedOnDark`.
- [ ] `borderHoverColor` renamed as `hoverBorderColor`.
- [ ] `borderHoverColorOnDark` renamed as `hoverBorderColorOnDark`.

**Comment by aweell** [comment]
Closed via #350
